db/schema.rb

```diff
@@ -36,6 +36,7 @@
     t.datetime "updated_at", null: false
     t.integer "source_id"
     t.jsonb "bootstrappers", default: [], null: false
+    t.string "container_type", default: "STATEFUL", null: false
```

Comment:
let's not put business logic in db,
If you need default, better put it in model

Reply:
If I have a field that only has 2 possible values: **stateful** or **stateless**. (can't be **null**)

How can I migrate the previous rows in the database? The database will throw an exception, said that the field can't be added because the database doesn't know things to put on the previous available rows.

Do you have any workaround for this?

Reply:
ok then you can add another migration that turns off default value after container_type column has been added

Reply:
Today I learned that I can do that. Thanks mas!